docs/source/documentation.rst

```diff
@@ -0,0 +1,117 @@
+Documentation Guidelines
```

Comment:
I did a very quick review of this file. I think it is on the right track, but should be more explicit in specifying the contents that are needed for the RST file within the `Analyzers` and `Machine Designs` sections of the table of contents.

Also... can we turn on section numbering here, like we do for the `Analyzers`?

<img width="902" alt="image" src="https://github.com/Severson-Group/eMach/assets/7421348/2b4594a7-c42d-4fc9-9ec7-0ed8b2cb2423">

Reply:
Let's discuss, I have implemented in the latest pull.

Reply:
I am resolving this conversation. The comments have been addressed and will be part of the next push to review.